src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCasePreviewServiceTest.java

```diff
@@ -348,7 +362,7 @@ public void willSetPreviewFileWithoutInterpreterDescription_WhenInterpreterRequi
         sscsCaseData.setAdjournCaseGenerateNotice("yes");
         sscsCaseData.setAdjournCaseTypeOfNextHearing(nextHearingType);
         sscsCaseData.setAdjournCaseNextHearingDateType("firstAvailableDate");
```

Comment:
These kinds of fields are the same on nearly every test, set them in in the method setUp() and then override when necessary, it makes the tests clearer in what you are testing.